packages/components/src/components/flyout/flyout.tsx

```diff
@@ -154,31 +143,28 @@ export class Flyout {
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
-      <Host onMouseDown={this.onMouseDown}>
+      <dialog
+        inert={this.open ? null : true} // prevents focusable elements during fade-out transition
+        tabIndex={-1} // dialog always has a dismiss button to be focused
+        ref={(ref) => (this.dialog = ref)}
+        onCancel={this.onCancelDialog}
+        onClick={this.onClickDialog}
+        {...parseAndGetAriaAttributes(this.aria)}
```

Comment:
Is this still fine here or should it go onto `.wrapper`?

Reply:
In my opinion this is the right place for the aria attributes